distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Split_Tokenize.enso

```diff
@@ -51,6 +53,44 @@ tokenize_to_rows table input_column_id pattern="." case_sensitivity=Case_Sensiti
     Value_Type.expect_text column
         fan_out_to_rows table input_column_id (handle_nothing (_.tokenize pattern case_sensitivity))
 
+parse_to_columns : Table -> Text | Integer -> Text -> Case_Sensitivity -> Boolean -> Problem_Behavior -> Table
+parse_to_columns table input_column_id pattern="." case_sensitivity=Case_Sensitivity.Sensitive parse_values=True on_problems=Report_Error =
+    case_insensitive = case_sensitivity.is_case_insensitive_in_memory
+    pat = Regex.compile pattern case_insensitive=case_insensitive
+
+    fun = handle_nothing (regex_parse_to_vectors pat)
+    column_namer = regex_to_column_namer pat input_column_id
+
+    column = table.at input_column_id
+
+    new_table = Value_Type.expect_text column <|
+        fan_out_to_rows_and_columns table input_column_id fun column_namer on_problems
+    if parse_values then new_table.parse on_problems=on_problems else new_table
+
+## PRIVATE
+   Create a parser from a regex to a nested `Vector`. Each match becomes an
+   element outer vector; each group (or the whole match, if there are no groups)
+   becomes an element of the inner vectors.
+regex_parse_to_vectors : Pattern -> (Text -> Vector (Vector (Text | Any)))
+regex_parse_to_vectors pattern =
+    input->
+        matches = pattern.match_all input
+        case pattern.group_count of
+            1 -> matches.map match-> [match.text]
+            _ -> matches.map match-> match.groups . drop 1
+
+## PRIVATE
+   Create a column namer that uses regex group names, if any.
+   If the regex has no explicit groups, it returns the original column name
+   unchanged.
+regex_to_column_namer : Pattern -> Text -> (Integer -> Text | Nothing)
+regex_to_column_namer pattern original_column_name =
+    case pattern.group_count of
+        1 -> _-> original_column_name
+        _ ->
+            group_nums_to_names = pattern.group_nums_to_names
+            i-> group_nums_to_names.get (i+1)  # explicit groups start at 1
```

Comment:
nitpick:
```suggestion
			# explicit groups start at 1
            i-> group_nums_to_names.get (i+1)
```

I think we tend to put the comments _above_ stuff, not _after_ that.

Maybe because with the old parser that line would likely crash 🤣